### PR TITLE
Rm delivery table

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

- Drops the delivery table

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b rm-delivery-table -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: alembic/versions/2024_03_27_9cf71c905539_remove_delivery_table.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+ª"""remove_delivery_table
```
Comment (diitaz93) [inline alembic/versions/2024_03_27_9cf71c905539_remove_delivery_table.py:1]: typo?
```suggestion
"""remove_delivery_table
```

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

**Comment by islean** [comment]
Test order placed successfully ✅

**Comment by islean** [comment]
Downgrade successful:

```
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/stage/bin/git/cg
done.
Log deploy... done.
cg, version 60.6.3
[isak.ohlsson@hasta:cg] [S_base] master ± 
```

Different charsets though 🤔 

Prod: 
image

Stage:
image

**Comment by islean** [comment]
Deployed to hasta and cg-services-prod:
```
 tests/meta/orders/test_meta_orders_status.py                      | 22 ++--------------------
 12 files changed, 51 insertions(+), 70 deletions(-)
 create mode 100644 alembic/versions/2024_03_27_9cf71c905539_remove_delivery_table.py
/home/isak.ohlsson
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 0fda0f2746d6 -> 9cf71c905539, remove_delivery_table
[isak.ohlsson@hasta:~] [P_base] $ 
```

**Pull request merged**